Mira raises KeyError: 'value'  when parsing BIOMD0000000949

I decided to go through all the Biomodels AMRs and see if I could load them, and got this mira error:

```python

../../mira/mira/sources/askenet/petrinet.py:106: in template_model_from_askenet_json
 
    param_values = {
>       p['id']: p['value'] for p in ode_semantics.get('parameters', [])
    }
E   KeyError: 'value'

../../mira/mira/sources/askenet/petrinet.py:107: KeyError
```

The problem appears to be that the AMR does not have  a value for the `baseline_dummy_variable` parameter:

https://github.com/DARPA-ASKEM/experiments/blob/953808422c72dd0ab91dc7114ff3e581dfef0dac/thin-thread-examples/mira_v2/biomodels/BIOMD0000000949/model_askenet.json#L339-L346

Even though the underlying SBML model does contain a value for the parameter

https://github.com/DARPA-ASKEM/experiments/blob/953808422c72dd0ab91dc7114ff3e581dfef0dac/thin-thread-examples/mira_v2/biomodels/BIOMD0000000949/src/main/Chitnis2008.xml#L853

--- a/mira/modeling/askenet/petrinet.py
+++ b/mira/modeling/askenet/petrinet.py
@@ -169,7 +169,7 @@ class AskeNetPetriNetModel:
             if param.placeholder:
                 continue
             param_dict = {'id': str(key)}
-            if param.value:
+            if param.value is not None:
                 param_dict['value'] = param.value
             if not param.distribution:
                 pass

--- a/mira/sources/askenet/petrinet.py
+++ b/mira/sources/askenet/petrinet.py
@@ -105,6 +105,7 @@ def template_model_from_askenet_json(model_json) -> TemplateModel:
 
     param_values = {
         p['id']: p['value'] for p in ode_semantics.get('parameters', [])
+        if p.get('value') is not None
     }
 
     # Next we process initial conditions

